recipes/data_bag.rb

```diff
@@ -30,7 +30,14 @@
 
 # only manage the subset of users defined
 Array(user_array).each do |i|
-  u = data_bag_item(bag, i.gsub(/[.]/, '-'))
+  name = i.gsub(/[.]/, '-')
+
+  u = if node['user']['data_bag_encrypted']
+    Chef::EncryptedDataBagItem.load(bag, name, node['user']['data_bag_encryption_key'])
```

Comment:
We should rely on the key being provided in the client's config and not here. We should remove any references to this attribute:

```Ruby
Chef::EncrytedDataBagItem.lad(bag, name)
```